PrimeNumbers/PrimeNumbers.cpp

```diff
@@ -20,4 +21,85 @@ bool PrimeNumbers::IsPrimeNumber(long& number)
 			return false;
 	}
 	return true;
+}
+
+bool PrimeNumbers::IsDivisibleBy(const long& total, const int division)
+{
+	return (total %division == 0) ? true : false;
+}
+
+string PrimeNumbers::ShowPrimeSequence(const long& numberOfPrimes)
+{
+	stringstream outputStream;
+	if (numberOfPrimes > 0)
+	{
+		GenerateNumberOfPrimes(numberOfPrimes);
+	}
+	if (m_PrimeNumbers.size() > 0)
+	{
+		for (auto n : m_PrimeNumbers)
+		{
+			outputStream << n;
+			if (n != m_PrimeNumbers.back())
+			{
+				outputStream << ", ";
+			}
+		}
+	}
+	return outputStream.str();
+}
+
+string PrimeNumbers::OutputPrimeFactors(const long& number)
+{
+	stringstream factors;
+	GeneratePrimesUpTo(number);
+	long total = number;
+	while (total > static_cast<long>(m_minimumNumber))
+	{
+		for (auto n : m_PrimeNumbers)
+		{
+			if ((n <= total) && (IsDivisibleBy(total, n)))
```

Comment:
`(n <= total)` can be made redundant. I'll leave it up to you to figure out how :P